build/yamls/windows/containerd-with-ovs/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -1,31 +0,0 @@
-$ErrorActionPreference = "Stop"
```

Comment:
May I know why you want to migrate this script to start_ovs_windows_containerd?

Reply:
Since , after adding steps related to installation of driver and other ovs related dependencies the Run-Antrea_OVS-Containerd.ps1 script was becoming too long , this script was moved to builds/images/scripts and added to antrea/windows image to improve readability of code. Any other suggestion is welcome.

Reply:
@NamanAg30 Wenying and I just discussed about this change, we prefer using the original script and focus on the driver logic changes in this PR to facilitate the code review. If you want, you can perform additional refactoring in a separate PR after new CI pipeline is available.